var/spack/repos/builtin/packages/yaml-cpp/package.py

```diff
@@ -39,11 +39,13 @@ class YamlCpp(Package):
 
     depends_on('cmake', type='build')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            args = std_cmake_args
-            if '+fpic' in spec: 
-                args += ['-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=true']
-            cmake('..', *args)
-            make()
-            make("install")
+    def cmake_args(self):
+        spec = self.spec
+        options = []
+
+        if '+fpic' in spec: 
+            options.extend([
```

Comment:
Why not use `options.append()` instead of `options.extend([])`? No need to create a new list.